app/models/user.rb

```diff
@@ -1,2 +1,8 @@
 class User < ApplicationRecord
+  has_many :posts
+  # has_many :comments, class_name: 'Comment'
+  # has_many :likes, class_name: 'Like'
+  def recent_post
+    posts.limit(3).order(created_at: :desc)
+  end
```

Comment:
- [ ] you are using the right code here, however, it is not in the right order. it should be something like `posts.order('created_at Desc').limit(3)`. the reason it should be like this is that in your code you are first taking 3 posts from the top of your list and after that, you are ordering these 3 posts by creation date. you should first order it to get the 3 most recent posts.